jaeger-core/src/main/java/com/uber/jaeger/reporters/RemoteReporter.java

```diff
@@ -59,12 +61,14 @@ public RemoteReporter(final Sender sender, int flushInterval, int maxQueueSize,
     commandQueue = new ArrayBlockingQueue<>(maxQueueSize);
 
     // start a thread to append spans
+    long serialNumber = nextSerialNumber.getAndIncrement();
```

Comment:
yes, you make the change in the same branch as you did and just push that branch, the PR should be automatically updated.